### DISCUSSION TITLE
How to edit split border colour.

### DISCUSSION THREAD

**Discussion by ChrisTaylorDeveloper** [catppuccin/nvim#676] [Q&A] [answered]
I find it very difficult to see the border between splits.  How can I edit the split border colour please?

**Comment by mrtnvgr** [comment, marked as answer]
You can overwrite `WinSeparator` highlight group:

```lua
require("catppuccin").setup {
    custom_highlights = function(colors)
        return {
            WinSeparator = { fg = colors.flamingo },
        }
    end
}
```

**Reply by ChrisTaylorDeveloper** [reply]
Perfect! Thank you.

**Reply by marques-bruno** [reply]
Sorry to bump a closed topic, but it seems to work for horizontal splits, however vertical splits don't seem to have borders.
Is there a way to change that? (add a horizontal line between splits)

**Reply by ChrisTaylorDeveloper** [reply]
I had some success with vim.opt.laststatus=3 which removes the status bar and therefore the horizontal split line becomes visible.  Combine this with Neovim 'win bar', which can take over some of the duties of the missing status bar.

**Reply by marques-bruno** [reply]
Thanks, that works perfectly! :)
